src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -137,7 +137,7 @@ cdef void plugin_get_metadata(
   cy_context.context = context
   self.plugin_callback(cy_context, python_callback)
 
-cdef void plugin_destroy_c_plugin_state(void *state):
+cdef void plugin_destroy_c_plugin_state(void *state) with gil:
```

Comment:
Sorry for the long round trip. What's up with the added GIL acquisition? Does the decref go boom without it? I could've sworn Cython was supposed to whine if acquiring the GIL was needed to avoid going boom.